idds_wrapper/tests/test_command_processor.cpp

```diff
@@ -0,0 +1,54 @@
+#include "gtest/gtest.h"
+#include "idds_wrapper/idds_common.h"
+#include "idds_wrapper/command_processor.h"
+
+using namespace testing;
+
+class CommandProcessorTest : public ::testing::Test {
+protected:
+    CommandProcessor commandProcessor;
+
+    void SetUp() override {
+        // Register a sample callback for testing
+        commandProcessor.registerCallback("TestCommand", [](const ParamList& params, std::string& response) {
```

Comment:
```suggestion
        commandProcessor.registerCallback("TestCommand", [](const ParamList& /* params */, std::string& response) {
```
some compilers can complain about unused variables